Cannot resolve universal promise on 0.19.2

After updating to the latest version, I get the following:

```
ERROR in ./~/react-formal/lib/errorManager.js
Module not found: Error: Cannot resolve module 'universal-promise' in /<my_project>/node_modules/react-formal/lib
```

--- a/package.json
+++ b/package.json
@@ -106,6 +106,7 @@
     "sinon-chai": "^2.8.0",
     "style-loader": "^0.13.1",
     "teaspoon": "^6.4.0",
+    "universal-promise": "^1.1.0",
     "url-loader": "^0.5.7",
     "webpack": "^1.13.1",
     "webpack-dev-server": "^1.14.1"

